core/src/main/java/feign/Client.java

```diff
@@ -59,6 +59,8 @@
 
   class Default implements Client {
 
+    private static final String HTTP_PROTOCOL_VERSION = "HTTP/1.1";
```

Comment:
To avoid/reduce impact on any clients out there, we should move this to response.... and make the default value for `protocolVersion` when unset.

So it's up for HTTP2 clients to update it.

Reply:
done